client/src/pages/OnBoarding/OnBoarding.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+  Paper,
+  Grid,
+  makeStyles,
+  Typography,
+  Divider,
+} from "@material-ui/core";
+import { Route, Switch, Redirect, useRouteMatch } from "react-router-dom";
+
+import SetTimezoneUrl from "./SetTimezoneUrl";
+import ConnectGoogleCalendar from "./ConnectGoogleCalendar";
+import SetAvailability from "./SetAvailability";
+import ProgressBar from "./smallComponents/ProgressBar";
+
+const OnBoarding = (props) => {
```

Comment:
By removing the state from the parent component, you're probably making your life a lot harder when you need to start making requests to the backend.

Reply:
I did think about it. But since each request sent to the backend would be slightly different, wouldn't it make more sense to have each child component handle their own state and request?